pkg/generator/templates.go

```diff
@@ -424,6 +424,27 @@ spec:
   version: {{.Version}}
 `
 
+const testYamlTmpl = `apiVersion: v1
+kind: Pod
+metadata:
+  name: {{.ProjectName}}-test
+spec:
+  restartPolicy: Never
+  containers:
+  - name: {{.ProjectName}}-test
+    image: {{.Image}}
+    imagePullPolicy: Always
+    command: ["/go-test.sh"]
+    resources:
+      requests:
+        cpu: 1
```

Comment:
I don't think we should specify resource requests by default. This could prevent the pod from being scheduled onto a node if there isn't enough cpu.
By not setting a default, the pod will either take on the default request and limit enforced by the namespace, or use as much as is available.